src/docker.py

```diff
@@ -57,6 +57,8 @@ def extract_resources_from_docker(self):
             self.__extract_from_docker('crustio/crust', '/opt/crust/crust')
         elif self.chain_name == 'subsocial':
             self.__extract_from_docker('dappforce/subsocial-parachain', '/usr/local/bin/subsocial-collator')
+        elif self.chain_name == 'robonomics':
+            self.__extract_from_docker('robonomics/robonomics', 'usr/local/bin/robonomics')
```

Comment:
Do you know if they have multiple chains, e.g. one relay and one para?

Just checking if this should be "robonomics-mainnet" or something.

Reply:
The only have one parachain, which is this one on the kusama network as at the time of this writing. So I guess this should be enough, if they do spin up another chain we could just modify this